app/controllers/repos_controller.rb

```diff
@@ -41,13 +41,26 @@ def create
     @repo   = Repo.search_by(params[:repo][:name], params[:repo][:user_name]).first unless params_blank?
     @repo ||= Repo.new(repo_params)
     if @repo.save
-      @repo_sub = current_user.repo_subscriptions.create(repo: @repo)
-      flash[:notice] = "Added #{@repo.to_param} for triaging"
-      RepoSubscription.background_send_triage_email(@repo_sub.id)
-      redirect_to @repo
+      @repo_sub = current_user.repo_subscriptions.find_or_initialize_by(repo: @repo)
```

Comment:
@prathamesh-sonpatki I chose to use find_or_initialize_by instead of using find_create_by so that I could flash the proper message after clicking the add repo button. The two scenarios that might happen are:

1) Flashing of "Added <repo name> for triaging"
2) Repository already exists

I did this so that the users will be properly informed.